packages/core/src/components/Table/types.ts

```diff
@@ -24,8 +25,10 @@ export interface TableColumnConfig {
     fraction?: number;
     /** Column content alignment */
     align?: 'left' | 'right' | 'center';
-    /** WIP | Set it true to freeze the component on the left side*/
+    /** WIP | Set it true to freeze the component on the left side */
```

Comment:
Do we need `WIP` now ? Also now the description could be `Set it true to freeze the column based on freezePosition`